### PR TITLE
:alien: feat(Kinto): configuration for kintohub

### PR DESCRIPTION
**Pull request by Disturbing**
* minor configurations on influx helm charts to work with future kintohub usage history

### CODE REVIEW

File: charts/telegraf-ds/templates/configmap.yaml [hunk 1]
```diff
@@ -38,3 +38,10 @@ data:
     url = "https://$HOSTIP:10250"
     bearer_token = "/run/secrets/kubernetes.io/serviceaccount/token"
     insecure_skip_verify = true
+    label_include = []
```
Comment (Disturbing) [inline charts/telegraf-ds/templates/configmap.yaml:41]: Note: this will include all labels

File: charts/telegraf-ds/templates/configmap.yaml [hunk 2]
```diff
@@ -38,3 +38,10 @@ data:
...
+
+    [[inputs.kube_inventory]]
+    url = "{{ .Values.kubernetes.apiHost }}"
+    bearer_token = "/run/secrets/kubernetes.io/serviceaccount/token"
+    insecure_skip_verify = true
+    resource_exclude = ["daemonsets", "deployments", "endpoints", "persistentvolumes", "persistentvolumeclaims", "services", "statefulsets" ]
```
Comment (Disturbing) [inline charts/telegraf-ds/templates/configmap.yaml:47]: Note: i don't think this is working, but can optimize later.

File: charts/telegraf-ds/templates/configmap.yaml [hunk 3]
```diff
@@ -38,3 +38,10 @@ data:
     url = "https://$HOSTIP:10250"
     bearer_token = "/run/secrets/kubernetes.io/serviceaccount/token"
     insecure_skip_verify = true
+    label_include = []
+
+    [[inputs.kube_inventory]]
+    url = "{{ .Values.kubernetes.apiHost }}"
+    bearer_token = "/run/secrets/kubernetes.io/serviceaccount/token"
+    insecure_skip_verify = true
```
Comment (nandiheath) [inline charts/telegraf-ds/templates/configmap.yaml:46]: comment: i see insecure and I want to know why oO

File: charts/telegraf-ds/templates/role.yaml [hunk 4]
```diff
@@ -14,6 +14,17 @@ rules:
...
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: influx:cluster:viewer
+  labels:
+    rbac.authorization.k8s.io/aggregate-view-telegraf: "true"
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes", "nodes"]
+    verbs: ["get", "list"]
```
Comment (nandiheath) [inline charts/telegraf-ds/templates/role.yaml:26]: comment: permission here is enough? how to avoid we miss the right permissions when we do the deploy?

**Pull request merged**